.github/workflows/swift-toolchain.yml

```diff
@@ -943,8 +951,6 @@ jobs:
           variant: sccache
 
       - name: Configure Compilers
-        env:
```

Comment:
I dont think that we can remove this

Reply:
Why? I think this env var `NDKPATH` was unused and the build worked without it.